sql/catalyst/src/main/scala/org/apache/spark/sql/types/StructField.scala

```diff
@@ -160,7 +164,7 @@ case class StructField(
    */
   def toDDL: String = {
     val nullString = if (nullable) "" else " NOT NULL"
-    s"${quoteIfNeeded(name)} ${dataType.sql}${nullString}$getDDLComment"
+    s"${quoteIfNeeded(name)} ${dataType.sql}${nullString}$getDDLDefault$getDDLComment"
```

Comment:
Just curious, what happens we have a multi-line default value string like `a\n b\n c\n d`?